### PR TITLE
Add file completion from classpath for @ContextConfiguration

### PR DESCRIPTION
**Pull request by ksankaranara-vmw**
Added content-assist feature to auto complete xml files from the classpath for @ContextConfiguration annotations
Issue: #650

### CODE REVIEW

File: headless-services/spring-boot-language-server/src/main/java/org/springframework/ide/vscode/boot/java/contextconfiguration/ContextConfigurationProcessor.java
Comment (martinlippert) [inline headless-services/spring-boot-language-server/src/main/java/org/springframework/ide/vscode/boot/java/contextconfiguration/ContextConfigurationProcessor.java:1]: The copyright for new files should be `Broadcom`, not `Pivotal` anymore.

File: headless-services/spring-boot-language-server/src/test/java/org/springframework/ide/vscode/boot/java/contextconfiguration/test/ContextConfigurationCompletionTest.java
Comment (martinlippert) [inline headless-services/spring-boot-language-server/src/test/java/org/springframework/ide/vscode/boot/java/contextconfiguration/test/ContextConfigurationCompletionTest.java:1]: The copyright for new files should be `Broadcom`, not `Pivotal` anymore.

**Comment by martinlippert** [comment]
Tried this out in more detail, looks great in general. Great work.

What I noticed (assuming that there is a file `root-config.xml`):

```
@ContextConfiguration(locations ="root<*>")
```

produces the correct proposals, whereas:

```
@ContextConfiguration(locations ="/root<*>")
```

does not produce any proposals. Same happens if there is a longer patch to the resource. It seems like the `/` at the beginning is breaking the logic.

In addition to that I also noticed that the short summary of the proposal doesn't really work well if there is a longer patch for the proposals. In those cases, most of the stuff that the proposal shows in the popup is a long patch, but the real name of the file is hidden (because the proposal label is too long for the popup). And the short version that shows up at the end of the selected proposal inside of the popup seems to repeat the label, so doesn't help either.

But this seems to be a general problem with path-based proposals, the `classpath:` proposals for `@Value` annotations have the same problem. There is probably some fine tuning necessary, but we should address that in a separate issue once this PR is merged.

But would be great to get the first part here (missing proposals if the prefix starts wit `/`) addressed before we merge.

**Comment by ksankaranara-vmw** [comment]
@martinlippert thanks for the review! I'll work on the changes.

**Review by BoykoAlex — APPROVED** [review]

**Comment by martinlippert** [comment]
@ksankaranara-vmw Looks like @BoykoAlex did already merge this PR, so would be great if you could submit an additional PR for the issue around missing proposals if the prefix starts with `/`... :-)

**Comment by ksankaranara-vmw** [comment]
@martinlippert sure :-)

**Pull request merged**
